test/katex-spec.js

```diff
@@ -849,13 +849,6 @@ describe("A color parser", function() {
         expect(newColorExpression).toParse();
     });
 
-    it("should have correct greediness", function() {
```

Comment:
I think there may be another reference to `greediness` in the `A style change parser` describe block.